server/lib/getScopes.js

```diff
@@ -4,11 +4,24 @@ import * as constants from '../constants';
 
 
 const getUserAccess = (user, type) => {
+  let namespace = null;
+
+  _.forEach(user, (val, key) => {
+    if (key.indexOf('https:') === 0 && key.indexOf('auth0-delegated-admin') > 0) {
```

Comment:
I think this could be simplified as:
```js
let namespace = _.find(user, (val, key) => _.startsWith(key, 'https:') && _.endsWith(key, 'auth0-delegated-admin'))
```

Or perhaps `endsWith` is too restrictive, and should just be `contains`?